action.yml

```diff
@@ -64,16 +64,12 @@ runs:
       shell: bash
       run: terminus auth:login --machine-token=${{ inputs.pantheon_machine_token }}
     - name: "Deploy to Pantheon environment ${{ inputs.pantheon_env_name }}"
+      # Skipped if autopromote is false or we're not deploying to the dev environment.
+      if: ${{ inputs.autopromote == 'true' && (inputs.pantheon_env_name == 'production' || inputs.pantheon_env_name == 'master' || inputs.pantheon_env_name == 'dev') }}
```

Comment:
I believe this would prevent all deployments to multisite environments

Reply:
It's deploying to a multidev right now for me.

Reply:
multidevs would have `autopromote` set to false.